@xen-orchestra/web-core/lib/components/state-hero/VtsObjectNotFoundHero.vue

```diff
@@ -1,13 +1,14 @@
 <template>
-  <VtsStateHero class="vts-object-not-found-hero" image="no-result" type="page">
+  <VtsStateHero class="vts-object-not-found-hero" image="no-result" :type>
     {{ $t('object-not-found', { id }) }}
   </VtsStateHero>
 </template>
 
 <script lang="ts" setup>
-import VtsStateHero from '@core/components/state-hero/VtsStateHero.vue'
+import VtsStateHero, { type StateHeroType } from '@core/components/state-hero/VtsStateHero.vue'
 
 defineProps<{
   id: string
+  type: StateHeroType
```

Comment:
Since you set the `type` prop as required in this component, remember to check the usages and modify accordingly.
I'm getting some errors when running `type-check`, so when making such changes in components, I think it is good practice to run `yarn type-check --watch` (in `/web` and `/lite` directories) to catch issues.
The same remark applies for the other `Vts*Hero` components that have been updated.